### PR TITLE
Chapter_recurrent-modern/seq2seq.md

### PR DESCRIPTION
**Pull request by zhuyuanxiang**

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-zh/PR-764/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-764/

File: chapter_recurrent-modern/seq2seq.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 #  序列到序列学习（seq2seq）
 :label:`sec_seq2seq`
 
-正如我们在 :numref:`sec_machine_translation` 中看到的。在机器翻译中，输入和输出都是可变长度的序列。为了解决这类问题，我们在 :numref:`sec_encoder-decoder` 中设计了一个通用的编码器-解码器结构。在本节中，我们将使用两个循环神经网络来设计此编码器-解码器结构，并将其应用于机器翻译 :cite:`Sutskever.Vinyals.Le.2014,Cho.Van-Merrienboer.Gulcehre.ea.2014` 的*序列到序列*（sequence to sequence）学习。
+正如我们在 :numref:`sec_machine_translation` 中看到的。在机器翻译中，输入序列和输出序列都是长度可变的。为了解决这类问题，我们在 :numref:`sec_encoder-decoder` 中设计了一个通用的”编码器－解码器“结构。在本节中，我们将使用两个 RNN 来设计此“编码器－解码器”结构，并将其应用于机器翻译 :cite:`Sutskever.Vinyals.Le.2014,Cho.Van-Merrienboer.Gulcehre.ea.2014` 的*序列到序列*（sequence to sequence）学习。
```
Comment (goldmermaid) [inline chapter_recurrent-modern/seq2seq.md:4]: Hi @zhuyuanxiang , could you replace RNN with "循环神经网络" in this article. I know it is a bit verbose, but this is the standard we are trying to maintain.

Comment (zhuyuanxiang) [reply chapter_recurrent-modern/seq2seq.md:4]: I have done it.
How about GRU and LSTM?

**Comment by mli** [comment]
Job d2l-zh/PR-764/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-764/

File: chapter_recurrent-modern/seq2seq.md (hunk 1 above)
Comment (goldmermaid) [reply chapter_recurrent-modern/seq2seq.md]: Great question! Please replace the corresponding Chinese terms as well. Thanks!

Comment (zhuyuanxiang) [reply chapter_recurrent-modern/seq2seq.md]: How about other files? Could I change them?
I think maybe we could add the term in `TERMINOLOGY.MD`, so we could have the uniform translation.

Comment (zhuyuanxiang) [reply chapter_recurrent-modern/seq2seq.md]: Excuse me, could you teach me what's the meaning about the "Outdated' label? Thks!

**Comment by mli** [comment]
Job d2l-zh/PR-764/4 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-764/

File: chapter_recurrent-modern/seq2seq.md (hunk 1 above)
Comment (goldmermaid) [reply chapter_recurrent-modern/seq2seq.md]: No worries about the "Outdated" label. It reminds us that the reference that this comments about might be "Outdated".

Comment (goldmermaid) [reply chapter_recurrent-modern/seq2seq.md]: Yes please the modify the other files, appreciate your effort! Also, could you help update the `terminology.md` file?

Comment (zhuyuanxiang) [reply chapter_recurrent-modern/seq2seq.md]: I have done it.

**Pull request merged**